manager/manager.go

```diff
@@ -509,6 +533,22 @@ func (m *manager) destroyContainer(containerName string) error {
 		})
 	}
 	glog.Infof("Destroyed container: %q (aliases: %v, namespace: %q)", containerName, cont.info.Aliases, cont.info.Namespace)
+
+	contRef, err := cont.handler.ContainerReference()
+	if err != nil {
+		return err
+	}
+
+	newEvent := &events.Event{
+		ContainerName: contRef.Name,
+		EventData:     cont,
```

Comment:
We also shouldn't use cont here. What data do you think would be useful here? Spec would be nice, but that's gone by now. Maybe the ContainerReference? That's just the full names of the container. Maybe just blank and leave as is?